src/Resources/skeleton/crud/templates/show.tpl.php

```diff
@@ -0,0 +1,28 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+    <h1><?= $entity_class_name; ?></h1>
+
+    <table>
+    <?php foreach ($entity_fields as $field): ?>
+        <tr>
+            <th><?= ucfirst($field['fieldName']); ?></th>
+            <td>{{ <?= $entity_var_singular; ?>.<?= $field['fieldName']; ?> }}</td>
+        </tr>
+    <?php endforeach; ?>
+    </table>
+
+    <ul>
```

Comment:
Just remove the `ul` and `li` tags. They are unnecessary and if there is debate about the styling, just keep it as minimal as possible (a `<br>` could be enough).

Reply:
agreed. They are really unnecessary.

Reply:
indeed, ul and li should be remove. However, br is not a good id (should not be use outside of `<p>`) I think a simple div would be enough
I also agree about a separate file for delete form. 
And about CSS, it should not be include (or with an option for included bootstrap classes maybe? not by default)